gateway/models/runbooks.go

```diff
@@ -190,3 +191,37 @@ func BuildCommonConfig(config *RunbookRepositoryConfig) (*commonRunbooks.Config,
 
 	return commonRunbooks.NewConfigV2(configInput)
 }
+
+func CreateDefaultRunbookConfiguration(db *gorm.DB, orgID string) (*Runbooks, error) {
+	const defaultRepoURI = "https://github.com/hoophq/demo-runbooks"
+	const defaultRepoName = "demo-runbooks"
```

Comment:
`repoName` must be `github.com/hoophq/demo-runbooks`, same applies for the migrations, etc.